docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
       - CODECOV_TOKEN=$CODECOV_TOKEN
       - NPM_AUTH_TOKEN=$NPM_AUTH_TOKEN
-      - DATABASE_URL=postgres://postgres:postgres@pg:5432/test
+      - DATABASE_URL=postgres://postgres:posgres@pg:5432/test
```

Comment:
what? isn't the password here a typo since the `t` is missing?

Reply:
that's actually what it is in `bluefin-link`. I'm just trying anything to see if I can get these tests passing on CI
https://github.com/dailymuse/bluefin-link/blob/develop/docker-compose.yml#L13

Reply:
Huh, interesting but if the password is wrong i think the password everywhere else won't work. Perhaps, fixing where the password is set will help.